cartographer_ros/cartographer_ros/offline_node.cc

```diff
@@ -152,8 +152,21 @@ void RunOfflineNode(const MapBuilderFactory& map_builder_factory) {
       },
       false /* oneshot */, false /* autostart */);
 
-  auto bag_expected_sensor_ids =
-      node.ComputeDefaultSensorIdsForMultipleBags(bag_trajectory_options);
+  std::vector<
+      std::set<cartographer::mapping::TrajectoryBuilderInterface::SensorId>>
+      bag_expected_sensor_ids;
+  if (configuration_basenames.size() == 1) {
+    bag_expected_sensor_ids = node.ComputeDefaultSensorIdsForMultipleBags(
```

Comment:
more readable: `auto expected_sensor_ids =` and then below you can use the n-elements vector constructor instead of insert.